src/components/CategoryFilter.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CATEGORIES = [
+  'All',
```

Comment:
- [ ] It would be better if we merge the “All” select option with the available categories. “All” is not a valid book category and shouldn’t be saved in the Redux store.